From: Matthias Klumpp <matthias@tenstral.net>
Date: Thu, 06 Aug 2015 13:18:02 +0000
Subject: Add AppStream metadata
X-Git-Url: http://quickgit.kde.org/?p=umbrello.git&a=commitdiff&h=70ecfc71c7bf6400560b964d8ed9dfaf2a6abb57
---
Add AppStream metadata

BUG:352607

(cherry picked from commit fa2014c9d3edb2cb144bc9f5f0edafa789c2975c)

Conflicts:
umbrello/CMakeLists.txt
---


--- a/umbrello/org.kde.umbrello.appdata.xml
+++ b/umbrello/org.kde.umbrello.appdata.xml
@@ -1,1 +1,33 @@
+<?xml version="1.0" ?>
+<component type="desktop">
+  <id>org.kde.umbrello.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <name>Umbrello</name>
+  <summary>UML Modeller</summary>
+  <description>
+    <p>
+      Umbrello is a Unified Modelling Language (UML) modelling tool and code generator.
+      It can create diagrams of software and other systems in the industry-standard UML format, and can also generate code from UML diagrams
+      in a variety of programming languages.
+    </p>
+    <p>Features:</p>
+    <ul>
+      <li>Supported formats: XMI</li>
+      <li>Several type of diagrams supported: use case, class, sequence, collaboration, state, activity, component, deployment, entity relationship</li>
+    </ul>
+  </description>
+  <url type="homepage">http://umbrello.kde.org</url>
+  <url type="bugtracker">https://bugs.kde.org/enter_bug.cgi?format=guided&amp;product=umbrello</url>
+  <url type="help">http://docs.kde.org/stable/en/kdesdk/umbrello/index.html</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>http://kde.org/images/screenshots/umbrello.png</image>
+    </screenshot>
+  </screenshots>
+  <project_group>KDE</project_group>
+  <provides>
+    <binary>umbrello</binary>
+  </provides>
+</component>
 

